docs/user/introduction.rst

```diff
@@ -37,7 +37,6 @@ The GeoMesa website may be found at http://www.geomesa.org/. For additional info
 * The GeoMesa FAQ: http://www.geomesa.org/faq/
```

Comment:
or we should probably just update the FAQs... @mdzimmerman do you know where that code lives? i can update it real quick